project-koku/koku
# [COST-2465] streamline ocp on cloud trino tag matching sql

Fixes [COST-2465](https://issues.redhat.com/browse/COST-2465)

Changes proposed in this PR:
* Move to using the OCP daily summary table in Trino for OCP on cloud tag matching
* Eliminate lookup on several tables
* Take into account enabled tag tables for Trino OCP on Cloud tag matching SQL
* Check for OCP enabled tags before doing tag matching SQL -- if a user has no enabled tags, there is nothing to match

Testing Instructions:
1. Run standard load test customer data script
2. Make sure we have data for OCP on AWS/Azure/GCP

---
Additional Context:
